pywinauto/controls/common_controls.py

```diff
@@ -657,6 +658,46 @@ def deselect(self):
     # Non PEP-8 alias
     Deselect = deselect
 
+    #-----------------------------------------------------------
+    def inplace_control(self, friendly_class_name=""):
+        """Return the editor HwndWrapper of the item
+
+        Possible ``friendly_class_name`` values:
+
+        * ``""``  Return the first appeared in-place control
+        * ``"friendlyclassname"``  Returns editor with particular friendlyclassname

```

Comment:
Just an idea for improvement for all controls: related global timings could be documented for every relevant method. It will simplify timings customization.